# This is done because, for example, OpenDNS has a fallback for non-existent hosts.
# 
# This behaviour would have given a 200 code, where the module would expect a 500.
#

--- a/t/10-nonexistent-host.t
+++ b/t/10-nonexistent-host.t
@@ -16,7 +16,7 @@ my $called;
   *WWW::Mechanize::Shell::status = sub {};
 };
 
-$s->cmd('get http://nonexistent.host');
+$s->cmd('get http://999.999.999.999');
 is($s->agent->res->code, 500, "LWP returns 500 for host not found");
 
 };
